### DISCUSSION TITLE
MADIC3S 2025 Pre-Meeting 1 2024-11-04

### DISCUSSION THREAD

**Discussion by s4b7r** [MADICES/MADICES-2025#5] [Announcements]
Placeholder for minutes, notes, follow-up tasks, etc. of the 1st MADICES 3 2025 pre-event meeting on 2024-11-04.

We are preparing our meeting minutes right now. @samikanza do we post the minutes here once they are ready?

**Comment by samikanza** [comment]
@s4b7r happy to stick minutes in here, should we make it an announcement rather than general though?

**Reply by s4b7r** [reply]
yes, making it an announcement would be nice but I think I dont have the permissions in the MADICES Github organization nor in this repo. I can't change the category to announcement. Do you have permissions? Otherwise maybe @edan-bainglass or @ml-evs can make us part of the organization or invite us properly to this repo? Would make sense for the future anyways, I guess.

**Reply by s4b7r** [reply]
I think we need to be repo maintainers for this.

**Reply by s4b7r** [reply]
Thanks to @ml-evs , adding us to the org worked and I could make this topic an announcement now.

**Comment by samikanza** [comment]
### **Meeting 1 - 04/11/2024**

**Date:** 4th November 2024

**Attendees (28)**

**Organisers:** Edan Bainglass, Caterina Barillari, Simon Bekemeier, Sebastian Brückner, Matthew Evans, Samantha Pearman-Kanza
Invitees: Abraham Nieva, Adam Laskowski, Andreas Meier, Brian Matthews, Burkhard Schӓfer, Carlo Minotti, Fabio Lopes, Hampus Nӓsström, Ivano Eligio Castelli, Jan Janssen, Juan Fuentes, Jeanne Wilbrandt, Jörg Schaarschmidt, Martin Starman, Mihai-Cosmin Danalia, Monika Vogler, Nukorn Plainpan, Peter Kraus, Sarah Callaghan, Sebastian Brückner, Steffen Brinckmann, Stefan Kuhn

**Purpose:** The key purpose of this initial kick off meeting was to introduce the new organising team, introduce the attendees, and provide a holistic overview of the topics/discussions/plans for MADICES 3

### **Discussion Items** 

**1. Organiser Introductions** 

<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-e47c7083-7fff-1b3c-6442-e87dbf983000"><div dir="ltr" style="margin-left:0pt;" align="left">
Name | Institute / Project | Interest/Description
-- | -- | --
Edan Bainglass | PSI, Switzerland <br>PREMISE | Was a co-organiser for MADICES 2. Developer of AiiDA/AiiDAlab/MaterialsCloud. Member of the PREMISE project. Interested in developing standards and protocols to facilitate interoperability of ORD across research platforms in a platform-agnostic manner.
Samantha Pearman-Kanza | University of Southampton, UK<br>PSDI | Joined for MADICES 2. Been working in RDM / ELNS / Semantics for the last 10 years. Very interested in semantic annotation, ELNS, RDM  and interoperability.
Simon Bekemeier | BAM (Bundesanstalt für Materialforschung), Germany<br>PMD (Plattform MaterialDigital) | joined for 2nd workshop in Berlin, working in a digitisation initiative for materials (Platform MaterialDigital) mainly interested in digital workflows and those that interface with the real world / data gathering, and taking unstructured data and converting into structured data - interoperability.
Caterina Barillari | ETH Zurich<br> PREMISE/openBIS | Responsible for data management services based on openBis (ELNs) and also involved in the PREMISE project. Was partly co-organiser for MADICES 2
Sebastian Brückner | FAIRMAT, NOMAD | FAIRMat project in the area of synthesis - working on the NOMAD project. Looking at digitisation of data management. Interested in interoperability.
Matthew Evans | UCLouvain<br>PSDI, DataLab/OPTIMADE/Data Tractor | MADICES 1 Organiser - research moving towards experimental research data management and creating data lab - interested in shared protocols across ELNS/LIMS, ELN interoperability, and research data lifecycle primarily across materials and chemistry - connect with other successful communities. Want to make sure we have some serious action at the workshop honed from our discussions.
</div></b>

**2. Attendee Introductions**

<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-4864d235-7fff-82eb-2bdc-f375430b32d4"><div dir="ltr" style="margin-left:0pt;" align="left">
Name | Institute / Project | Interest/Description
-- | -- | --
Abraham Nieva | University of Cardiff / PSDI | Workflows (Nextflow, Galaxy workflows) - 10 ROCrates which can be imported into the galaxy instance hosted in STFC (Materials Galaxy) - interested in interoperability between different workflow systems
Adam Laskowski | ETH Zurich, PREMISE | Working as openBIS developer - platform for open reproducible sciences, working on interoperability
Andreas Meier | ETH Zurich | openBIS developer, working on interoperability with RO-Crate and RDF
Brian Matthews | STFC/PSDI | Interest in tools and services to support data archiving, data interoperability, data exchange and data re-use across the physical sciences - background in data management and data metadata for neutron sources - working on data sharing and publication
Burkhard Schӓfer | Splashlake<br>AniML/SILA | Active in the area of standardisation and interoperability - interested in bridging gaps between different systems and bridging the last mile towards the instrument - interested in the content generated from the experiment and instruments - how do we model experiments that use multiple pieces of equipment / cross organisation - need to put things in context, metadata is key for this
Carlo Minotti | PSI | Works on the part of project trying to standardise data catalogues. Interested in interoperability between different platforms (e.g. ELN/data catalogues).
Fabio Lopes | EMPA<br>PREMISE | Part of the nanotech group, works on the PREMISE project - works on interoperability between different management systems - AIIDA Lab - ELN (openBis) - use their lab (nanotech/nanomaterials) as a use case for this interoperability
Hampus Nӓsström | FAIRmat | Works at FAIRMAT Berlin, producing a FAIR data research platform for materials data, want to figure out how we can apply the semantics and proposed interoperability for real world usage by scientists e.g. schemas/formats
Ivano Eligio Castelli | DTU Energy | Works on data infrastructure and availability of methods and workflows, works with simulations and bridge with machine learning and experiments
Jan Janssen | Max Planck Institute for Sustainable Materials | CONDA Forge project - scientific software / materials scientific software for the wider community, interested in connecting on a workflow level, and in sustainable software and accessible software. Has worked on atomistic simulations.
Juan Fuentes | ETH Zurich | Technical lead of openBIS development team - established in interoperability / RO-Crate
Jeanne Wilbrandt | Leibniz Institute on Aging | Data Steward in Germany - Bio/Zoology - working in the human aspect - convincing researchers to document their research in their ELNs and not needlessly copy information into many different places. Wants to see how we can use this to help people and be attractive to people.
Jörg Schaarschmidtt | Karlsruhe Institute of Technology (KIT) | Working with Simon on materials data project - works with workflow management tools, interested in simulation of workflows
Martin Starman | Karlsruhe Institute of Technology (KIT)Chemotion | Works in FAIR Data, ELNs, and bridging data across ELNs
Mihai-Cosmin Danalia | PREMISE | Software developer, working on UI development for this project. Has participated in MADICES 2 before.
Monika Vogler | HIU Germany | Working in batteries and lab automation, interested in robotics and how to get the data out of the robot and into a database for use in ML / get scientific insights from the data
Nukorn Plainpan | EMPA | Working in the PREMISE project, working on research data, battery development, involves developing tools for researchers to ontologise their metadata, and using openBIS
Peter Kraus | TUBData Tractor (with Matthew Evans), https://yard.datatractor.org/ | Wants to get lab automation software to work with some of the ELNS and workflow managers using the protocols they are currently developing
Sarah Callaghan | University of Oxford | Research Practice Manager - Oxfords research culture programme - supporting researchers to do their research better, in a more open, transparent , and reliable way. Primarily focusing on training researchers to do the right things / best practice, moving onto supporting them to build and develop tool and services which will make it easy for them to do the right thing re research practice
Steffen Brinckmann | Forschungszentrum Jülich | Develops PASTA ELN - started the ELN File Format / RO-Crate - exchanging between different ELNS, have 5-6 implementations currently.
Stefan Kuhn | Tartu University | chem. Informatics: interest in standards, interoperability, getting data into shape for “learning in general”, not only ML

</div></b>

**3. Online Resources for MADICES**
- GitHub Site - https://madices.github.io/ 
- Top Level GitHub Repo - https://github.com/MADICES/ 
- Discussions https://github.com/MADICES/MADICES-2025/discussions - Feel free to add to these
- Workshop Report from MADICES 2024 - https://madices.github.io/docs/2024/report/ 

**4. Overall Topics / Core Discussions from previous events - These formed the Working Groups for MADICES 2024** 
- [Platform-agnostic interoperability](https://github.com/MADICES/MADICES-2024/discussions/7) - platforms (workflow management systems, databases, ELNs, archives) - how do we move things around between all of these things in some form of standard but without forcing yet another standard / How can we move data around in the most interoperable way possible 
- [Semantic Annotation](https://github.com/MADICES/MADICES-2024/discussions/9) - how can this assist in providing the full picture of the datasets in order to reason and make sense of it as it is ported between different systems
- [Proprietary Datasets](https://github.com/MADICES/MADICES-2024/discussions/11) - How can we handle proprietary data and deal with the different standards 
- Looking to further these discussions and also work on new areas

**5. New Topics for Working Groups**
- [Standardisation and best practices](https://github.com/MADICES/MADICES-2025/discussions/1)
- [Integration of new technologies (AI/ML)](https://github.com/MADICES/MADICES-2025/discussions/3)
- [Use-case testing of tools and protocols](https://github.com/MADICES/MADICES-2025/discussions/2)
- [Community outreach and education](https://github.com/MADICES/MADICES-2025/discussions/4) 
- We encourage everyone to contribute to these discussions 

**6. Discussion points**
- We need to be mindful of Including the human element in this work, this is not just a technology driven endeavour. 
- How do we share our developments e.g. transformation scripts and little pieces of work to help with our data, we need a process to share these with the community so we aren’t constantly repeating the same bits of work. E.g. taking data in a raw state and then converting it into more structured data. Transforming unstructured data into structured data
- Key goal of MADICES - interoperability between different workflows 

**7. MADICES 3 Format**
- Hackathon oriented days
- Plenty of time for discussion
- Day 1 - potentially a panel / formal discussion (this was quite an effective approach at FLL 2024)
- Details on travel/funding/meals will emerge early next year as we find out about funding 
- Note that other projects that people are already doing can be integrated and incorporated as and when - place to try different approaches - link to existing work and feel free to bring along your own work 

**8. Next Pre Meeting**
- Aim for concrete working groups - split members into these groups and clearly define our goals / construct set of concrete deliverables 
- Aim to have these every 2-3 months (to be discussed internally) - next meeting will probably be in January - Doodle poll to be sent out soon

**9. Actions for next pre meeting** 
- Look at website (Everyone) 
- Read Reports (Everyone) 
- Contribute to GitHub Discussions & Repos (or add new topics) (Everyone) 
- Look at the tools that have been mentioned on the repos / by people in this meeting (Everyone) 
- Email out all the links sent in todays chat (Organisers)
- Organise dates for February Meeting  (Organisers)
